multiscanner/web/app.py

```diff
@@ -111,9 +112,17 @@ def system_health():
 
 
 def _main():
-    app.run(debug=app.config.get('DEBUG', DEFAULTCONF['DEBUG']),
-            port=app.config.get('PORT', DEFAULTCONF['PORT']),
-            host=app.config.get('HOST', DEFAULTCONF['HOST']))
+    in_docker = str(os.getenv("PRODUCTION", False))
+
+    if in_docker == "True":
+        from gevent.pywsgi import WSGIServer
+        http_server = WSGIServer((app.config.get('HOST', DEFAULTCONF['HOST']),
+        app.config.get('PORT', DEFAULTCONF['PORT'])), app, log=sys.stdout)
```

Comment:
Just curious, why are log messages are being passed to `sys.stdout`? If needed, should we do the same for `api.py`?